project-koku/koku
# COST-2892: adding invoice month to GCP summary tasks

## Jira Ticket

[COST-2892](https://issues.redhat.com/browse/COST-2892)

## Description 

Add GCP invoice month to summary tasks. This is because we miss month bound crossover data when running summary via standard dates. With this change we can pass in start/ends past a month boundary with the inclusion of invoice month so the full billed month is summarised rather than just a date range. 

## Testing

1. Checkout Branch
2. Restart Koku
3. Add real GCP data
4. Verify all the numbers add up correctly per month

## Notes

...
